ccpp/suites/suite_HAFS_v0_hwrf.xml

```diff
@@ -37,12 +37,14 @@
     </subcycle>
     <!-- Surface iteration loop -->
     <subcycle loop="2">
-      <scheme>sfc_diff</scheme>
+      <scheme>gfdl_sfc_layer</scheme>
```

Comment:
what is the difference between the sfc_dff and the gfdl_sfc_layer?

Reply:
I defer this question to @grantfirl who added the GFDL surface layer scheme. As far as I understand, it is entirely different from the GFS surface layer scheme.

Reply:
Yes, it is the surface layer scheme from WRF called the "GFDL surface layer scheme" used in HWRF. It is not closely related to sfc_diff. It's been a long time since I worked on this and don't remember many of the details, but you're welcome to look at the PR when this was originally merged into the HWRF CCPP physics: (https://github.com/NCAR/ccpp-physics/pull/452)